### PR TITLE
FIRE-3246 clean up console warnings

### PR DESCRIPTION
**Pull request by pfmartins-zartis**
# <span title="Add the ticket link below"> Ticket :information_source: </span>
[FIRE-3246](https://605tv.atlassian.net/browse/FIRE-3246)
# <span title="One or two lines, non technical"> Description :information_source: </span>
Update the Month component to properly set unique keys for its children.
# <span title="Detailed, technical"> Summary :information_source: </span>

# <span title="Screenshots, video recording, before <> after"> How to Test :information_source: </span>
- Clone this repo (the local repo folder **must be named** as `605-react-date-range`);
- in this repo: 
   -  `yarn install`
   - `yarn build`
   - `yarn link`
- in platf0rm client repo:
  - `yarn add link:../../605-react-date-range`
  - go to any page with a DatePicker;
  - no warnings of duplicate or missing key from react should appear

[FIRE-3246]: https://605tv.atlassian.net/browse/FIRE-3246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: src/components/Month/index.js [hunk 1]
```diff
@@ -31,18 +31,18 @@ function renderWeekdays(styles, dateOptions, weekdayDisplayFormat, broadcastCale
...
+              <span className={styles.weekDay}>
                 {format(day, weekdayDisplayFormat, dateOptions)}
               </span>
-            </>
+            </Fragment>
           );
         }
         return (
-          <span className={styles.weekDay} key={i}>
+          <span className={styles.weekDay} key={`day-cell-${i}`}>
```
Comment (matheus-gomes-dev) [inline src/components/Month/index.js]: [We have more than one Month component rendered in the DOM](https://github.com/605data/react-date-range/blob/e617c90c57688ba3e1f19e9c377af0871df0130f/src/components/Calendar/index.js#L496), if only the `i` index is used to define the key value, I think we'd still have duplicated keys for the week days cells. Maybe we should use the month value here as well to define the key?

File: src/components/Month/index.js [hunk 2]
```diff
@@ -31,18 +31,18 @@ function renderWeekdays(styles, dateOptions, weekdayDisplayFormat, broadcastCale
       }).map((day, i) => {
         if (i === 0 && broadcastCalendar) {
           return (
-            <>
-              <span className={styles.weekDay} key='week-number'>
+            <Fragment key={`week-number-${i}`}>
```
Comment (matheus-gomes-dev) [inline src/components/Month/index.js]: Please see comment below

**Review by matheus-gomes-dev — CHANGES_REQUESTED** [review]

File: src/components/Month/index.js (hunk 1 above)
Comment (erpankajsingla) [reply src/components/Month/index.js]: @matheus-gomes-dev what is the scenario to reproduce this? I couldn't reproduce. 
Also, I couldn't get this working using yarn link but I used as below instead. And I don't see any errors anymore. 
```
"605-react-date-range": "git+https://github.com/605data/react-date-range.git#61bee906b85524405ae9be38711fc1bdfdb51854",
```

File: package.json [hunk 3]
```diff
@@ -46,8 +46,7 @@
   "peerDependencies": {
     "date-fns": "2.0.0-alpha.7 || >=2.0.0",
     "react": "^0.14 || ^15.0.0-rc || >=15.0"
-  },
-  "devDependencies": {
+  }, "devDependencies": {
```
Comment (erpankajsingla) [inline package.json]:
```suggestion
  }, 
  "devDependencies": {
```

**Review by matheus-gomes-dev — APPROVED** [review]

**Pull request merged**